docs/spec/callables.rst

```diff
@@ -424,22 +424,59 @@ Meaning of ``...`` in ``Callable``
 
 The ``Callable`` special form supports the use of ``...`` in place of the
 list of parameter types. This indicates that the type is consistent with
-any input signature::
+any input signature. Just as ``Any`` means "any conceivable type that could be
+compatible", ``(...)`` means "any conceivable set of parameters that could be
+compatible"::
 
     cb1: Callable[..., str]
     cb1 = lambda x: str(x)  # OK
     cb1 = lambda : ""  # OK
 
     cb2: Callable[[], str] = cb1  # OK
 
-The input signature ``(...)`` is not the same as ``(*args: Any, **kwargs: Any)``.
-The former is a gradual type form that is consistent with all input signatures,
-and all input signatures are consistent with it. The latter is not a gradual
-type form. It is consistent with all input signatures, but the converse is
-not true. The relationship between ``(...)`` and ``(*args: Any, **kwargs: Any)``
-is similar to the relationship between ``Any`` and ``object``. Just as ``Any``
-means "any conceivable type that could be compatible", ``(...)`` means "any
-conceivable set of parameters that could be compatible".
+A ``...`` can also be used with ``Concatenate``. In this case, the parameters
+prior to the ``...`` are required to be present in the input signature and
+be compatible in kind and type, but any additional parameters are permitted::
+
+    cb3: Callable[Concatenate[int, ...], str]
+    cb3 = lambda x: str(x)  # OK
+    cb3 = lambda a, b, c: str(a)  # OK
+    cb3 = lambda : ""  # Error
+    cb3 = lambda *, a: str(a)  # Error
+
+
+If the input signature in a def statement includes both a ``*args`` and
```

Comment:
def statement -> function definition ?

Reply:
That would be better since one could argue that an `async def` is not a def statement.
```suggestion
If the input signature in a function definition includes both a ``*args`` and
```